src/Migration.php

```diff
@@ -547,7 +547,7 @@ public function dropField($table, $field)
 
 
     /**
-     * Drop immediatly a table if it exists
+     * Drop immediately a table if it exists
```

Comment:
New method should be used here, instead of a raw query.

Reply:
Done